### PR TITLE
EditorToolbar: add "transclusion" and "wikilink" buttons

### PR DESCRIPTION
**Pull request by BurningTreeC**
this adds two buttons to the editor toolbar to wrap selection into [[ ]] or {{ }}
they work like the existing toggle buttons

### CODE REVIEW

**Comment by Jermolene** [comment]
Hi @BurningTreeC that's great, thank you.

I'd rather we used images for consistency with the existing toolbar buttons. I think I've actually done both of these already, I'll dig into my archive.

**Comment by BurningTreeC** [comment]
@Jermolene 
yes that's great , I tried making them but I have other strengths :laughing: 

I've added ((transclusion)) and ((wikilink)) shortcuts but I'm not going to add them, just for if someone wants to it's already there

**Comment by BurningTreeC** [comment]
@Jermolene 
I've updated the buttons and the naming in buttons.multitids

I think it's ready

File: core/images/transclusion.tid [hunk 1]
```diff
@@ -0,0 +1,3 @@
+title: $:/core/images/transclusion
```
Comment (Jermolene) [inline core/images/transclusion.tid]: Hi @BurningTreeC I think we can lose this and `$:/core/images/wikilink`?

**Comment by Jermolene** [comment]
Thanks @BurningTreeC

**Comment by Jermolene** [comment]
Hi @BurningTreeC I've just noticed that the PR doesn't include the `$:/tags/EditorToolbar`
 tag on `$:/core/ui/EditorToolbar/linkify` and `$:/core/ui/EditorToolbar/transcludify`, is that an oversight?

**Comment by BurningTreeC** [comment]
@Jermolene 
yes it's an oversight
I've added the tags in my branch

**Pull request merged**